Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -130,10 +130,10 @@ public struct DocumentID<Value: DocumentIDWrappable & Codable>:
       service: "[FirebaseFirestoreSwift]",
       code: "I-FST000002",
       message: """
-      Attempting to initialize or set a @DocumentID property with a non-nil
-      value: \(value). The document ID is managed by Firestore and any
-      initialized or set value will be ignored. The ID is automatically set
-      when reading from Firestore."
+      Attempting to initialize or set a @DocumentID property with a non-nil \
+      value: \(value). The document ID is managed by Firestore and any \
```

Comment:
nit/optional: Should we add quotes around value? It could be helpful if the developer sets an empty string, for example.